### PR TITLE
[INLONG-2028][CI] Add support for docker build on GitHub Actions

### PR DESCRIPTION
**Pull request by shink**
Fixes: #2028 

### Motivation

Add support for docker build on GitHub Actions.

### Modifications

- `.github/workflows/ci_build_docker.yml`

### Verifying this change

Execute the following command.

```shell
mvn --batch-mode --update-snapshots -e -V clean verify -DskipTests -Pdocker
```

### CODE REVIEW

**Comment by dockerzhang** [comment]
@shink `InLong Docker Build`  workflow executes failed; please create an issue, I will fix it first.

File: .github/workflows/ci_build_docker.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+on:
+  push:
+    branches: [ master, 'INLONG-*' ]
+  pull_request:
+    branches: [ master, 'INLONG-*' ]
+
+jobs:
+  build:
+    name: Docker Build
+    runs-on: ubuntu-latest
```
Comment (dockerzhang) [inline .github/workflows/ci_build_docker.yml:29]:
```suggestion
    runs-on: ubuntu-latest
    env:
     CI: false
```

Comment (shink) [reply .github/workflows/ci_build_docker.yml:29]: Solved.

**Review by dockerzhang — APPROVED** [review]
LGTM

**Review by gosonzhang — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

**Pull request merged**